packages/extension-sdk-react/src/components/ExtensionProvider/ExtensionProvider.tsx

```diff
@@ -70,6 +70,7 @@ export const ExtensionProvider: React.FC<ExtensionProviderProps> = ({
         coreSDK: core31SDK,
```

Comment:
when do you think this will switch to `core40SDK`? Do we need a deprecation schedule for that?

Reply:
Yeah, we should probably deprecate ExtensionProvider in favor of ExtensionProvider2. What's the process for doing that in open source? Do we need google approval? Do we just mark it as deprecated?

Reply:
I've created ExtensionProvider40 in a separate PR. When that it published I will update all of the examples to use it. After that we can mark ExtensionProvider and ExtensionProvider2 as deprecated. Not what the process is. I can update the release notes for extension sdk react to indicate that ExtensionProvider40 is the preferred provider and that the other two will be deprecated but supported for at least a year. Or we can keep ExtenionProvider but mark coreSDK31 and coreSDK properties as deprecated. c